code/src/layouts/app-header/components/categories/Categories.constants.ts

```diff
@@ -0,0 +1 @@
+export const categories = ["Shop All", "Computers", "Tablets", "Mobile"];

```

Comment:
Please be aware, that in the future we will most likely use array of object, that will include `label` and `image` and maybe other fields, like in `Shop By Category` section on our website. So I think i would be good topic for discussion with our teammates. So you can prepare your component to this array of objects

Reply:
> Please be aware, that in the future we will most likely use array of object, that will include `label` and `image` and maybe other fields, like in `Shop By Category` section on our website. So I think i would be good topic for discussion with our teammates. So you can prepare your component to this array of objects

It's not related to my tasks. Only `ui` part
You can take and show us how it should be implemented.
I guess will be interesting